a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -0,0 +1,250 @@
+import test from 'ava';
+import {
+  agops,
+  agoric,
+  bankSend,
+  getUser,
+  openVault,
+  adjustVault,
+  closeVault,
+  getISTBalance,
+  getContractInfo,
+  ATOM_DENOM,
+  USER1ADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+import { getBalance } from './test-lib/utils.js';
+
+test.serial('attempt to open vaults under the minimum amount', async t => {
+  const activeVaultsBefore = await agops.vaults('list', '--from', USER1ADDR);
+  await bankSend(USER1ADDR, `20000000${ATOM_DENOM}`);
+  t.log('active vaults before:', activeVaultsBefore);
+
+  const mint = '3.0';
+  const collateral = '5.0';
+  const error = await t.throwsAsync(() =>
+    openVault(USER1ADDR, mint, collateral),
+  );
+
+  t.true(
+    error?.message.includes(
+      'Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}',
+    ),
+    'Error message does not contain the expected text',
+  );
```

Comment:
The form I'm more used  to seeing is this

```suggestion
  await t.throwsAsync(
    openVault(USER1ADDR, mint, collateral),
    message: 'Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}',
  );
```

There are forms that use regexps with `//` pattterns, as well.

Reply:
I missed the earlier suggestion regarding the use of regular expressions made by @dckc on this [comment](https://github.com/Agoric/agoric-sdk/pull/10123#discussion_r1772309612) for the error message. I wasn't aware that a regular expression could be provided as the expectation in t.throwsAsync(), which is why I overlooked it initially.

I also apologize for prematurely resolving the conversation. I understand that it made it harder to track the changes made and open questions. Moving forward, I will make sure to leave conversations open and wait for the original commenter to resolve them.

Thank you for your patience and guidance.

Changes addressed on commit [2f7d52d](https://github.com/Agoric/agoric-sdk/pull/10123/commits/2f7d52d4d4d293c1287bcdc4a36f86d49ffc46e4)